src/Microsoft.Dnx.Runtime/NuGet/SemanticVersion.cs

```diff
@@ -16,9 +16,6 @@ public sealed class SemanticVersion : IComparable, IComparable<SemanticVersion>,
         public SemanticVersion(string version)
             : this(Parse(version))
         {
-            // The constructor normalizes the version string so that it we do not need to normalize it every time we need to operate on it. 
-            // The original string represents the original form in which the version is represented to be used when printing.
-            OriginalString = version;
```

Comment:
It was sort of a UI feature. It was apparently confusing when `Install-Package Foo 1.0` would install `Foo (1.0.0)`. We probably shouldn't care about it keeping it around.